Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -1008,31 +1032,52 @@ def get_email_activity_data(
     top = args.get(TOP, EMPTY_STRING)
     select = args.get(SELECT, EMPTY_STRING).split(",")
     query_op = args.get(QUERY_OP, EMPTY_STRING)
+    fetch_all = args.get(FETCH_ALL, FALSE)
+    fetch_max_count = int(args.get(FETCH_MAX_COUNT, 5000))
```

Comment:
lets use `defaultvalue` in the `.yml` file here aswell

Reply:
done.